src/styles/reset.scss

```diff
@@ -1,116 +1,64 @@
-    display: revert;
-}
-
-/* Preferred box-sizing value */
+/* 1. Use a more-intuitive box-sizing model */
 *,
 *::before,
 *::after {
     box-sizing: border-box;
 }
-
-/* Fix mobile Safari increase font-size on landscape mode */
-html {
-    -moz-text-size-adjust: none;
-    -webkit-text-size-adjust: none;
-    text-size-adjust: none;
-}
-
-/* Reapply the pointer cursor for anchor tags */
-a,
-button {
-    cursor: revert;
-}
-
-/* Remove list styles (bullets/numbers) */
-ol,
-ul,
-menu,
-summary {
-    list-style: none;
-}
-
-/* For images to not be able to exceed their container */
-img {
-    max-inline-size: 100%;
-    max-block-size: 100%;
-    /* to avoid the gap created by a space char between inline elements */
-    vertical-align: bottom;
-}
-
-/* removes spacing between cells in tables */
-table {
-    border-collapse: collapse;
-}
-
-/* Safari - solving issue when using user-select:none on the <body> text input doesn't working */
+/* 2. Remove default margin */
+* {
+    margin: 0;
+}
+body {
+    /* 3. Add accessible line-height */
+    line-height: 1.5;
+    /* 4. Improve text rendering */
+    -webkit-font-smoothing: antialiased;
+}
+/* 5. Improve media defaults */
+img,
+picture,
+video,
+canvas,
+svg {
+    display: block;
+    max-width: 100%;
+}
+/* 6. Inherit fonts for form controls */
 input,
-textarea {
-    -webkit-user-select: auto;
-}
-
-/* revert the 'white-space' property for textarea elements on Safari */
-textarea {
-    white-space: revert;
-}
-
-/* minimum style to allow to style meter element */
-meter {
-    -webkit-appearance: revert;
-    appearance: revert;
-}
-
-/* preformatted text - use only for this feature */
-:where(pre) {
-    all: revert;
-    box-sizing: border-box;
-}
-
-/* reset default text opacity of input placeholder */
-::placeholder {
-    color: unset;
-}
-
-/* fix the feature of 'hidden' attribute.
-   display:revert; revert to element instead of attribute */
-:where([hidden]) {
-    display: none;
-}
-
-/* revert for bug in Chromium browsers
-   - fix for the content editable attribute will work properly.
-   - webkit-user-select: auto; added for Safari in case of using user-select:none on wrapper element*/
-:where([contenteditable]:not([contenteditable='false'])) {
-    -moz-user-modify: read-write;
-    -webkit-user-modify: read-write;
+button,
+textarea,
+select {
+    font: inherit;
+}
+/* 7. Avoid text overflows */
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
     overflow-wrap: break-word;
-    -webkit-line-break: after-white-space;
-    -webkit-user-select: auto;
 }
-
-/* apply back the draggable feature - exist only in Chromium and Safari */
-:where([draggable='true']) {
-    -webkit-user-drag: element;
+/* 8. Improve line wrapping */
+p {
+    text-wrap: pretty;
 }
-
-/* Revert Modal native behavior */
-:where(dialog:modal) {
-    all: revert;
-    box-sizing: border-box;
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    text-wrap: balance;
+}
+/* 9. Create a root stacking context */
+#root {
```

Comment:
Shouldn't this be in the index.scss? This an application-specific element ID and styling.

Reply:
Originally came with the reset, we prefer to keep it here for now.

Reply:
It doesn't make sense here.

Reply:
I think Josh makes a good case for it to be in the reset in his explanation: https://www.joshwcomeau.com/css/custom-css-reset/#nine-root-stacking-context-10

Quote from site: 
Root stacking context:
This last one is optional. It's generally only needed if you use a JS framework like React.
As we saw in [“What The Heck, z-index??”(opens in new tab)](https://www.joshwcomeau.com/css/stacking-contexts/), the isolation property allows us to create a new stacking context without needing to set a z-index.
This is beneficial since it allows us to guarantee that certain high-priority elements (modals, dropdowns, tooltips) will always show up above the other elements in our application. No weird stacking context bugs, no z-index arms race.
You should tweak the selector to match your framework. We want to select the top-level element that your application is rendered within. For example, create-react-app uses a `<div id="root">`, so the correct selector is `#root`.

Reply:
He's making a case for adding `isolation: isolate` to the root element, but he's not making a case for putting it in `reset.css`

We already have styles defined for this element, and I just wanted to group `isolation: isolate` with those styles here:

https://github.com/codux-templates/e-commerce-reclaim/blob/0c1446127802a417b09a76ad0b7df9225c9393c1/src/components/site-wrapper/site-wrapper.module.scss#L1-L5

Relying on application-specific element IDs in the reset.css seems wrong, but fine, keep it here.

Reply:
I admit you've convinced me, we'll change it.